addons/l10n_nl_edi/models/account_edi_format.py

```diff
@@ -82,11 +82,9 @@ def _export_nlcius(self, invoice):
     ####################################################
 
     def _check_move_configuration(self, invoice):
-        res = super()._check_move_configuration(invoice)
+        errors = super()._check_move_configuration(invoice)
         if self.code != 'nlcius_1':
-            return res
-
-        errors = []
+            return errors
```

Comment:
this diff is not related to this task, right?

Reply:
See https://github.com/odoo/odoo/pull/70679#discussion_r680974735